### PR TITLE
Issue fixed #188  Changed the footer background

### LINKED ISSUE #188: Add background to the Footer
Add a suitable background to the footer such that it looks separated from the other section of the webpage .

### PR DESCRIPTION
**Pull request by GOVINDDIXIT**
Issue fixed #188  Changed the footer background
capture

https://github.com/opencode18/OpenCode-Collaborative/issues/188

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -241,7 +241,7 @@ <h1>Projects</h1>
   </section>
 
   <!-- Footer -->
-  <footer id="footer" role="contentinfo">
+  <footer id="footer" role="contentinfo" style="background-color: #f4511e">
```
Comment (Anupam-dagar) [inline index.html]: use a the linear gradient used in cards

**Comment by GOVINDDIXIT** [comment]
captureddddd


Done

**Comment by Anupam-dagar** [comment]
change the direction of gradient similar to one present in cards, also change footer text color to white.

**Comment by GOVINDDIXIT** [comment]
opencode
UPDATED

**Comment by Anupam-dagar** [comment]
@pradeepgangwar @aswinzz suggestion on color? seems fine.

**Comment by pradeepgangwar** [comment]
Text looks dull ... also can we increase height a little bit. And @GOVINDDIXIT post live link in PR description.

**Comment by GOVINDDIXIT** [comment]
I think the issues was just to change the footer background and I hope I had done it well , enough for 10 points

**Comment by pradeepgangwar** [comment]
@Anupam-dagar U read that comment ^

**Comment by Anupam-dagar** [comment]
@GOVINDDIXIT it's not that you are limited strictly to the content in an issue.We can't merge if a PR give rise to new issues.If a PR is like that then it's not a good PR.

**Comment by GOVINDDIXIT** [comment]
footer height incresed
[link](https://govinddixit.github.io/OpenCode-Collaborative/)

**Comment by gauthamzz** [comment]
Lol what’s happening over here 😂😂

**Pull request merged**